### PR TITLE
Migrating test_concurrent_set.py from glusto to redant

### PR DESCRIPTION
**Pull request by nishith-vihar**
Migrating [test_case](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_concurrent_set.py) from glusto to redant

Updates: #292
Fixes: #420 
Signed-off-by: Nishith Vihar Sakinala nsakinal@redhat.com

### CODE REVIEW

**Comment by nishith-vihar** [comment]
The part where is_core_file_created function is used is commented. That part will be added wen the is_core_file_created function is implemented

**Comment by nishith-vihar** [comment]
```
vol_create_test-dist PASS

test_concurrent_set-dist PASS

vol_destroy_test-dist PASS

Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   1   |    100.0     |
|   Disruptive  |   0   |      0       |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 vol_create_test
 
+-------------+-------------+-------------------+
| Volume Type | Test Result |  Time taken (sec) |
+-------------+-------------+-------------------+
|     dist    |     PASS    | 8.357531547546387 |
+-------------+-------------+-------------------+

 test_concurrent_set
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     dist    |     PASS    | 344.12043952941895 |
+-------------+-------------+--------------------+

 vol_destroy_test
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     dist    |     PASS    | 10.836886405944824 |
+-------------+-------------+--------------------+



Framework runtime : 366.65723419189453
```

File: tests/functional/glusterd/test_concurrent_set.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        ret2 = redant.wait_till_async_command_ends(proc2)
+
+        if ret1['error_code'] != 0:
+            raise Exception("Concurrent volume set on different volumes "
+                            "simultaneously failed")
+        if ret2['error_code'] != 0:
+            raise Exception("Concurrent volume set on different volumes "
+                            "simultaneously failed")
+
+        ret = redant.check_core_file_exists(self.server_list, timestamp)
```
Comment (srijan-sivakumar) [inline tests/functional/glusterd/test_concurrent_set.py:80]: Nit: we can simplify this as
```
if redant.check_core_file_exists(self.server_list, timestamp):
   raise Exception("foo bar")
redant.logger.info("No core file found")
```

**Review by srijan-sivakumar — APPROVED** [review]
LGTM...maybe the nit can be taken in practice from next test case. Good to go ahead and merge this.

File: tests/functional/glusterd/test_concurrent_set.py (hunk 1 above)
Comment (nishith-vihar) [reply tests/functional/glusterd/test_concurrent_set.py:80]: Ack

**Pull request merged**